CHANGELOG.mdx

```diff
@@ -117,14 +117,14 @@ https://github.com/RasaHQ/rasa/tree/main/changelog/ . -->
   want to extract an additional slot, add the slot's name to the list of `required_slots`
   and add a method `extract_<slot name>` to your action:
 
-  ```python
+```python
   from typing import Text, Dict, Any, List, Optional
 
   from rasa_sdk.forms import (
-      FormValidationAction,
+      ValidationAction,
   )
 
-  class FormWithSlotExtractions(FormValidationAction):
+  class FormWithSlotExtractions(ValidationAction):
```

Comment:
Is this update to superclass necessary since we're preserving `FormValidationAction`?

Reply:
Just updated it.
`FormValidationAction` needs to be used for forms; `ValidationAction` for slots outside forms.

Reply:
Oh got it, you're right